src/annotations/actions/thunks.ts

```diff
@@ -23,3 +35,15 @@ export const writeThenFetchAndSetAnnotations = (
 
   fetchAndSetAnnotations()(dispatch)
 }
+export const deleteAnnotations = annotation => async (
+  dispatch: Dispatch<AnnotationAction | PublishNotificationAction>
+) => {
+  try {
+    await deleteAnnotation(annotation)
+    dispatch(deleteAnnotationAction(annotation))
+    dispatch(notify(deleteAnnotationSuccess()))
+    fetchAndSetAnnotations()(dispatch)
+  } catch (error) {
```

Comment:
let's cut this whole `catch` block out and put it in the presentation component.

so the body of the function would be something like:

```ts
export const deleteAnnotations = annotation => async (
  dispatch: Dispatch<AnnotationAction | PublishNotificationAction>
) => {
  await deleteAnnotation(annotation)
  dispatch(deleteAnnotationAction(annotation))
  fetchAndSetAnnotations()(dispatch)
}